models/rate_neuron_ipn.h

```diff
@@ -74,15 +74,18 @@ or
 
 This template class needs to be instantiated with a class
 containing the following functions:
- - input (nonlinearity that is applied to the input, either psi or phi)
- - mult_coupling_ex (factor of multiplicative coupling for excitatory input)
- - mult_coupling_in (factor of multiplicative coupling for inhibitory input)
 
-The boolean parameter linear_summation determines whether the input function
+- input (nonlinearity that is applied to the input, either psi or phi)
+- mult_coupling_ex (factor of multiplicative coupling for excitatory input)
+- mult_coupling_in (factor of multiplicative coupling for inhibitory input)
```

Comment:
These are the objects from the equation, do they need `:math:` or a number of back-ticks?
```suggestion
- `input` (nonlinearity that is applied to the input, either psi or phi)
- `mult_coupling_ex` (factor of multiplicative coupling for excitatory input)
- `mult_coupling_in` (factor of multiplicative coupling for inhibitory input)
```